### ISSUE TITLE
Make multiple-label a best-practice rule

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#855]
Instead of having the multiple-label check be part of the label rule, we should have a separate best-practice rule. Having a duplicate label **may be** a violation of WCAG, but it doesn't have to be.

**Comment by jkodu** [comment]
@WilcoFiers 
Was looking to pick this up, but there is no `duplicate-label` check, did you mean `multiple-label`?

**Comment by jkodu** [comment]
PR Merged. Closing issue.
